applications/registration-react/src/redux/rootReducer.js

```diff
@@ -1,24 +1,24 @@
 import { combineReducers } from 'redux';
 import { reducer as formReducer } from 'redux-form';
 import catalog from './modules/catalog';
-import datasets from './modules/datasets';
+import { datasetsReducer } from './modules/datasets';
 import catalogs from './modules/catalogs';
-import apis from './modules/apis';
-import apiFormStatus from './modules/api-form-status';
+import { apisReducer } from './modules/apis';
+import { apiFormStatusReducer } from './modules/api-form-status';
 import inputLanguage from '../components/language-picker/redux/reducer';
 import { datasetFormStatus } from './modules/dataset-form-status';
-import { apiCatalog } from './modules/apiCatalogs';
+import { apiCatalogReducer } from './modules/apiCatalogs';
 import { referenceDataReducer } from './modules/referenceData';
 
 const rootReducer = combineReducers({
   form: formReducer,
   catalog,
-  datasets,
+  datasets: datasetsReducer,
   catalogs,
-  apis,
-  apiFormStatus,
+  apis: apisReducer,
+  apiFormStatus: apiFormStatusReducer,
   datasetFormStatus,
-  apiCatalog,
+  apiCatalog: apiCatalogReducer,
```

Comment:
All of this could have been avoided with default exports

Reply:
funksjon er funksjon og modul er modul, jeg vil ikke blande de konseptene :)